plugins/client/io.ktor/client-sse/3.0.0-beta-1/documentation.md

```diff
@@ -0,0 +1,13 @@
+Client "Server-sent Events" (SSE) plugin for consuming events sent using the 
+[SSE specification](https://html.spec.whatwg.org/multipage/server-sent-events.html).
```

Comment:
You could also refer to the docs: https://ktor.io/docs/3.0.0-beta-1/sse-client.html